mock-recipe-server/api_index.html

```diff
@@ -148,11 +148,9 @@
         let url = new URL(`/${testcase.dataset.name}/api/v1`, window.location);
         testcase.querySelector('.url').textContent = url.href;
         testcase.querySelector('.browser-code').value = (
-          `Cu.import('resource://shield-recipe-client/lib/Storage.jsm');\n` +
-          `Storage.clearAllStorage();\n` +
-          `Services.prefs.setCharPref("extensions.shield-recipe-client.api_url", "${url.href}");\n` +
+          `Cu.unload('resource://shield-recipe-client/lib/RecipeRunner.jsm');\n` +
```

Comment:
I don't think that addresses the issue. Somehow, despite us unloading the modules when the add-on is shut down for an update, the modules we import into the browser console are the versions from the old add-on until we restart. I can't quite figure out why.